File: sprint2/By lessons/lesson10code1/lesson10code1.go
package main

import "fmt"

func func1(step int) {
    fmt.Println("Call me. Step:", step)
}

func sayHello(name string) {
    fmt.Printf("Привет, %s\n", name)
    printHoroscope(name)
    fmt.Printf("Пока, %s, хорошего дня!\n", name)
}

func printHoroscope(name string) {
    fmt.Printf("%s! Сегодня подходящий день для прогулок в парке и изучения рекурсии\n", name)
}

func main() {
    func1(1)
    func1(2)
    func1(3)

    sayHello("Гоша")
}

File: sprint2/By lessons/lesson10code1/lesson10code1.swift
func funcCall(step: Int) {
    print("Call me. Step:", step)
}

funcCall(step: 1)
funcCall(step: 2)
funcCall(step: 3)

func sayHello(name: String) {
    print("Привет, \(name)")
    printHoroscope(name: name.uppercased())
    print("Пока, \(name), хорошего дня!")
}

func printHoroscope(name: String) {
    print("\(name)! Сегодня подходящий день для прогулок в парке и изучения рекурсии")
}

sayHello(name: "Гоша")
